### ISSUE TITLE
[ACF] BootStrap from shared

### ISSUE THREAD

**Issue by DavidWells** [inboundnow/landing-pages#217]
Should this be pointing to the hardcoded shared lp path? https://github.com/inboundnow/landing-pages/blob/master/classes/class.acf-integration.php#L25

Using INBOUNDNOW_SHARED_PATH instead

So acf pulls from latest cta/shared?

Or do you have plans to pull out the whole acf-integration into shared?

**Comment by atwellpub** [comment]
I originally planned to bootstrap from shared. 

If it loads from shared we can used the INBOUNDNOW_SHARED_PATH constant.

I was having issues with the fields registering correctly when loading from shared so I pulled it out of shared and decided to bootstrap from the landing pages core class and still reference shared. 

I later found out that it was not an issue with bootstrapping from /shared/ but a different issue that was causing the fields not to load. I don't remember what the issue was but it got solved. So it might could be bootstrapped from shared again. 

But also bootstrapping from core with the hardcoded shared path shouldn't harm anything either. 

If Pro is loading ACF then the lite version of ACF won't be bootstrapped at all from any of the core plugins. If any of the core plugins load from shared 1 time then the others will not attempt to bootstrap it twice.
